interfaces/builtin/openvswitch_support.go

```diff
@@ -30,8 +30,8 @@ openvswitch
 // NewOpenvSwitchSupportInterface returns a new "openvswitch-support" interface.
 func NewOpenvSwitchSupportInterface() interfaces.Interface {
 	return &commonInterface{
-		name:              "openvswitch-support",
-		connectedPlugKMod: openvswitchSupportConnectedPlugKmod,
-		reservedForOS:     true,
+		name: "openvswitch-support",
+		connectedPlugKModModules: []string{"openvswitch"},
```

Comment:
Same as earlier, can you please just change that variable above and use the variable here. We can optimise this later but it will make the review more straightforward.